scripts/dev.js

```diff
@@ -41,14 +44,19 @@ const dev = async (maybeInit) => {
 
   const buildDLL = require('./buildDll')()
   const clearRedis = redis.flushall()
-  const migrateDB = require('./migrate')()
+  const migrateRethinkDB = require('./migrate')()
+  const pgConfig = {
```

Comment:
@tiffanyhan these prop names were totally undocumented in the package, can't blame ya for not seeing them! should be good to go now though. this sets up the pattern to call pgm from code